raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -152,6 +156,7 @@ def subdispatch_to_initiatortransfer(
     )
 
     if sub_iteration.new_state is None:
+        print("No new state, payment trask for initiator ends")
```

Comment:
I don't fully understand this message.

Reply:
I think is a typo it should say "No new state, payment task for initiator ends" since if you don't have a new state on the sub iteration the flow should end, or at least is what i understood.

Reply:
Yeah sorry about the typo.

I need to change this to use a logger